modules/addRemove.js

```diff
@@ -0,0 +1,24 @@
+const bookTitle = document.querySelector('#bookTitle');
+const bookAuthor = document.querySelector('#bookAuthor');
+const bookList = document.querySelector('#bookList');
+
+export const addBooks = (Books) => {
+  bookList.innerHTML = '';
+  for (let i = 0; i < Books.length; i += 1) {
```

Comment:
[OPTIONAL] It would be great idea if you redirect user to list page after creating a new book!